public/lessons/responsive_layout/flexbox_layout.md

```diff
@@ -1,6 +1,6 @@
 # Flexbox
 
-<img src="centered-flexbox.png" width="480px"/>
+<img src="/images/centered-flexbox.png" width="480px"/>
```

Comment:
@Habenzy This may also be broken, cannot confirm within the the review panel of Github. 😏

Reply:
This one is fine too.